tasks/task_manager.hh

```diff
@@ -272,14 +274,14 @@ public:
             static future<std::vector<task_identity>> get_children(module_ptr module, task_id parent_id);
         public:
             virtual task_group get_group() const noexcept = 0;
-            virtual future<bool> contains(tasks::task_id task_id) const = 0;
+            virtual future<std::optional<virtual_task_hint>> contains(tasks::task_id task_id) const = 0;
```

Comment:
Lets' add a doc for the this function for the difference between empty virtual_task_hint and std::nullopt.